nodes/common/pulp_node/error.py

```diff
@@ -1,5 +1,13 @@
 from gettext import gettext as _
 
+CLI_DEPRECATION_WARNING = \
```

Comment:
Is this what I am looking for?

https://github.com/pulp/pulp/blob/master/pulp.spec#L616